src/pages/Accounts/types.ts

```diff
@@ -0,0 +1,9 @@
+import type { Component } from 'vue';
+
+export interface TableAsset {
+  symbol: string
+  name: string
+  price: number
+  amount: number
+  icon: Component | null
+}
```

Comment:
You see, Iroha doesn't support `symbol` and `price`. It is a discrepancy between the design and reality. I would ask to stick to reality in the first place, making as many steps towards the design as possible, but to know where to stop. 

You can refer to this schema in the docs: [`AssetDefinition`](https://hyperledger.github.io/iroha-2-docs/reference/data-model-schema.html#assetdefinition). Although it is outdated, it still reflects what actual assets look like in Iroha.
 
And it is still reflected in `dto.ts`, expect the `logo` field. Since in Iroha it is stored as IPFS path, I don't know yet how it's going to work in the explorer. Resolved by the backend? Resolved on the front-end part? Idk, let's leave a stub for now.

TL;DR: remove `TableAsset`, rely only on data from DTOs.